src/common/cmdparse.cc

```diff
@@ -138,13 +138,20 @@ dump_cmd_to_json(Formatter *f, uint64_t features, const string& cmd)
 
   stringstream ss(cmd);
   std::string word;
+  bool positional = true;
 
   while (std::getline(ss, word, ' ')) {
+    if (word == "--") {
```

Comment:
IIUC, so you're defining a new convention here, like
```
mgr module enable \
name=module,type=CephString \
-- \
name=force,type=CephBool,req=false
```
so one can
```console
ceph mgr module enable selftest --force
```

could you add a simple sample in the comment explaining the syntax?